app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -59,12 +59,10 @@
 }
 
 .gem-c-image-card__title {
-  @include govuk-font(19, $weight: bold);
```

Comment:
This could be left in if you'd like - unlike using `@extend` with GOV.UK Frontend, the mixins are weightless so are included in the `govuk_frontend_support` Sass file.

Reply:
Good to know. The reason I took it out though is it's redundant - the govuk-heading class now gets appended onto the element, which provides all these styles (in fact, if I leave it in, it conflicts with the govuk-heading class, so better to remove it).